src/components/QuizzQuestion.js

```diff
@@ -65,75 +119,155 @@ const QuizzQuestion = ({
+              direction="column"
+              justify="space-aroundflex-start"
+              alignItems="stretch"
+              key={text}
+              className={clsx({
+                [classes.answer]: true,
+                [classes.valid]: isAnswered && valid,
+                [classes.invalid]: isAnswered && !valid,
+                [classes.validUserChoice]: givenAnswer === index && valid,
+                [classes.invalidUserChoice]: givenAnswer === index && !valid,
+              })}
+            >
+              <Button
+                variant="outlined"
+                onClick={() => dispatch({
+                  type: 'ANSWER',
+                  payload: { id, theme, index, valid },
+                })}
+              >
+                {text}
+              </Button>
+            </Grid>
+          ))}
+        </Grid>
+        <Grid
+          container
+          direction="column"
+          justify="center"
+          alignItems="center"
+        >
+          <Grid item xs={12} md={9} xl={8}>
+            <Card className={classes.card} elevation={6}>
+              <CardContent>
+                <Typography variant="h3" component="h2">
+                  {t('Explanation')}
+                </Typography>
+
+                <Typography
+                  variant="body2"
+                  dangerouslySetInnerHTML={{ __html: explanation.slice(350) }}
+                />
+              </CardContent>
+              <CardActions className={classes.card__actions}>
+                <Button size="small">{t('Learn More')}</Button>
+              </CardActions>
+            </Card>
+          </Grid>
+
+        </Grid>
+        <Grid
+          container
+          className={classes.quiz__nav_buttons}
+          direction="column"
+          justify="center"
+          alignItems="center"
+          spacing={2}
         >
-          <Button
-            variant="contained"
-            onClick={() => dispatch({
-              type: 'ANSWER',
-              payload: { id, theme, index, valid },
-            })}
-            // disabled={isAnswered}
+          <div className={classes.progress}>
+            {questions.map(({ id: qId, fields: { slug } }) => (
+              <span key={qId}>
+                <Link to={`/quizz/${theme}/${slug}`}>
+                  {qId === id
+                    ? (
+                      <div className={clsx({
+                        [classes.progress__bullet]: true,
+                        [classes.progress__bullet_answered]: isAnswered,
+                        [classes.progressbullet_answered_valid]: isAnswered,
+                        [classes.progress__bullet_answered_invalid]: isAnswered,
+                      })}
+                      />
+                    )
+                    : (
+                      <div className={clsx({
+                        [classes.progress__bullet]: true,
+                      })}
+                      />
+                    )}
```

Comment:
```jsx
                  {qId === id && (
                      <div className={clsx({
                        [classes.progress__bullet]: true,
                        [classes.progress__bullet_answered]: isAnswered,
                        [classes.progressbullet_answered_valid]: isAnswered,
                        [classes.progress__bullet_answered_invalid]: isAnswered,
                      })}
                      />
                  )}

                  {qId !== id && (
                      <div className={clsx({
                        [classes.progress__bullet]: true,
                      })}
                  )}
```


*(No ternary for multi-line JSX)*

Reply:
Also, this could be simplified to:
```jsx
                      <div className={clsx({
                        [classes.progress__bullet]: true,
                        [classes.progress__bullet_answered]: isAnswered && qId === id,
                        [classes.progressbullet_answered_valid]: isAnswered && qId === id,
                        [classes.progress__bullet_answered_invalid]: isAnswered && qId === id,
                      })}
                      />
```

Reply:
done